docker/stable/ubuntu14.04/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:trusty
+MAINTAINER Andrew Schwartzmeyer <andschwa@microsoft.com>
+
+ARG POWERSHELL_RELEASE=v6.0.0-alpha.9
+ARG POWERSHELL_PACKAGE=powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb
+
+# Setup the locale
+ENV LANG en_US.UTF-8
+ENV LC_ALL $LANG
+RUN locale-gen $LANG && update-locale
+
+# Install dependencies and clean up
+RUN apt-get update \
```

Comment:
These dependencies are taken straight from the .NET Core packages' dependencies. Some *may* be able to be pruned (for instance, I already pruned LTTng).